src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs

```diff
@@ -33,7 +33,7 @@ public static T[] ArrayOfUniqueValues<T>(int count)
             // of random-sized memory by BDN engine
             T[] result = new T[count];
 
-            var random = new Random(Seed); 
```

Comment:
Some leading and trailing white-space and embedded spaces after casts were in this file. The VSCode save cleaned those up to match the project code style, but I can revert the white-space changes if desired.

Reply:
No objections, resolving.